howtos/release-procedure.md

```diff
@@ -21,20 +21,51 @@ The build process for [publish-prod](https://github.com/camunda/camunda-platform
 
 ## Perform a minor release
 
-Minor releases to Camunda Platform 8 happen twice a year in April and October.
+Minor releases to Camunda Platform 8 happen twice a year in April and October, and the documentation is versioned on the same cadence.
 
-To prepare for a minor release, you'll need to create a new verison.
+To prepare for a minor release, you'll need to create a new version.
 
 ### Create new version
 
-Technically, the current contents are frozen in `docs` and copied to `versioned_docs` with the corresponding version. The process can be triggered by this Docusaurus command:
+The versioning process copies a snapshot of the current documentation from the un-versioned source to a new versioned source location.
 
-```bash
-npm run docusaurus docs:version 8.1
+Because we are using [multiple docusaurus docs instances](./versioning.md#instances-docs-vs-optimize), we technically create _two_ new versions at time of product release: one for the main documentation, and one for the Optimize documentation.
```

Comment:
```suggestion
Because we are using [multiple Docusaurus docs instances](./versioning.md#instances-docs-vs-optimize), we technically create _two_ new versions at time of product release: one for the main documentation, and one for the Optimize documentation.
```

Nitpick